configapp/static/js/live.js

```diff
@@ -0,0 +1,85 @@
+function Live(latest_json, latest_img, image_suffix)
+{
+    this.canvas = document.getElementById('livecanvas');
+    this.ctx = this.canvas.getContext("2d");
+    this.rect = this.canvas.getBoundingClientRect();
+
+    this.prediction = latest_json;
+    this.IMAGE_SUFFIX = image_suffix;
+
+    this.image = new Image();
+    this.image.src = '/image/captured/' + latest_img;
+    var _this = this;
+    this.image.onload = function() {
+        _this.redraw();
+    };
+}
+
+Live.prototype.redraw = function() {
+    if (this.image !== null) {
+        this.ctx.clearRect(0, 0, this.canvas.width, this.canvas.height);
+        this.ctx.drawImage(this.image, 0, 0);
+        
+        var obj = this.prediction['spots'];
+        for (var i = 0; i < obj.length; i++) {
```

Comment:
I was under the impression that this drawing is pretty much same as in Labeler. Could it be reused? 

Put differently, could the code in Labeler be refactored so that it could be used here?